util/awsservice/cloudwatchmetrics.go

```diff
@@ -113,6 +113,40 @@ func GetMetricData(metricDataQueries []types.MetricDataQuery, startTime, endTime
 	return data, nil
 }
 
+func GetMetricStatistics(metricName, namespace string, dimensions []types.Dimension,
```

Comment:
nit: this could be simplified to
```
metricStatsInput := cloudwatch.GetMetricStatisticsInput{
	MetricName: aws.String(metricName),
	Namespace:  aws.String(namespace),
	StartTime:  aws.Time(startTime),
	EndTime:    aws.Time(endTime),
	Period:     aws.Int32(periodInSeconds),
	Dimensions: dimensions,
}
if extendedStatType == nil {
    metricStatsInput.Statistics = statType
} else {
    metricStatsInput.ExtendedStatistics = extendedStatType
}

return CwmClient.GetMetricStatistics(ctx, &metricStatsInput)
```